conbench/entities/commit.py

```diff
@@ -21,6 +21,8 @@ class Commit(Base, EntityMixin):
     __tablename__ = "commit"
     id = NotNull(s.String(50), primary_key=True, default=generate_uuid)
     sha = NotNull(s.String(50))
+    branch = Nullable(s.String(100))  # github max chars: 255
```

Comment:
@alistaire47 wrote this—Ed, do you have a reason for the 100 char limit?

Reply:
I spent a little time trying to figure out how many characters a branch name can be, but it seems like it's system-dependent. Looking more, [it sounds like the _de facto_ GitHub limit is 244 characters](https://stackoverflow.com/questions/24014361/max-length-of-git-branch-name#comment122975637_24014513), though users may run into restrictions before that—various people [try to keep it much shorter](https://medium.com/leantaas-engineering/why-are-we-limiting-git-branch-name-length-to-28-characters-c49cb5f4ff9a).

In practice, I doubt we'll see much over 50, but 244 or 255 are reasonable values because of GitHub. Note that all of that is just the branch name, though; we moved to `{user}:{branch}` to handle forks, so somebody could conceivably have an exceedingly long username as well and blow it out; I'm not sure what GitHub's limit there is.

Reply:
I'm getting flashbacks to a prior job where we had a whole string of "surely someone won't make a variable name that's longer than 50 characters!" 😱 

I wonder if 255*2 would be a reasonable enough limit that we won't need to go back and fix it later